pipelines/datasets/br_cgu_servidores_executivo_federal/flows.py

```diff
@@ -0,0 +1,502 @@
+    )
+    materialize_after_dump = Parameter(
+        "materialize after dump", default=True, required=False
+    )
+    dbt_alias = Parameter("dbt_alias", default=True, required=False)
+
+    rename_flow_run = rename_current_flow_run_dataset_table(
+        prefix="Dump: ", dataset_id=dataset_id, table_id=table_id, wait=table_id
+    )
+
+    next_date = get_next_date()
+
+    data_is_up_to_date = is_up_to_date(next_date, upstream_tasks=[next_date])
+
+    with case(data_is_up_to_date, True):
+        log_task("Tabelas estão atualizadas")
+
+    with case(data_is_up_to_date, False):
+        log_task(f"Starting download, {next_date}, {next_date}")
+        sheets_info = download_files(
+            date_start=next_date, date_end=next_date, upstream_tasks=[next_date]
+        )
+        log_task("Files downloaded")
+
+        data_clean_by_table = merge_and_clean_data(
+            sheets_info, upstream_tasks=[sheets_info]
+        )
+        log_task("Data clean finished")
+
+        outputs_path_by_table = make_partitions(
+            data_clean_by_table, upstream_tasks=[data_clean_by_table]
+        )
+        log_task("Partitions done")
+
+        with case(
+            table_is_available(outputs_path_by_table, "cadastro_aposentados"), True
+        ):
+            wait_upload_table_aposentados_cadastro = create_table_and_upload_to_gcs(
+                data_path=outputs_path_by_table["cadastro_aposentados"],
+                dataset_id=dataset_id,
+                table_id="cadastro_aposentados",
+                dump_mode="append",
+                wait=outputs_path_by_table,
+            )
+
+        with case(
+            table_is_available(outputs_path_by_table, "cadastro_pensionistas"), True
+        ):
+            wait_upload_table_pensionistas_cadastro = create_table_and_upload_to_gcs(
+                data_path=outputs_path_by_table["cadastro_pensionistas"],
+                dataset_id=dataset_id,
+                table_id="cadastro_pensionistas",
+                dump_mode="append",
+                wait=outputs_path_by_table,
+            )
+
+        with case(
+            table_is_available(outputs_path_by_table, "cadastro_servidores"), True
+        ):
+            wait_upload_table_servidores_cadastro = create_table_and_upload_to_gcs(
+                data_path=outputs_path_by_table["cadastro_servidores"],
+                dataset_id=dataset_id,
+                table_id="cadastro_servidores",
+                dump_mode="append",
+                wait=outputs_path_by_table,
+            )
+
+        with case(
+            table_is_available(
+                outputs_path_by_table, "cadastro_reserva_reforma_militares"
+            ),
+            True,
+        ):
+            wait_upload_table_reserva_reforma_militares_cadastro = (
+                create_table_and_upload_to_gcs(
+                    data_path=outputs_path_by_table[
+                        "cadastro_reserva_reforma_militares"
```

Comment:
Boa tarde Pedro! Vi que tu fez somente um flow pra todas as tabelas. De modo geral não vejo problema e tem o ponto positivo de evitar repetição. 

Tenho um comentário: qual é o tamanho da carga total do conjunto de tabelas que será atualizado a cada mês?

1. Pelo que entendi são 7 tabelas clusterizadas que vão atualizar todo mês. Algumas delas grandes (ex. cadastro_servidores tem 145240600 linhas)
2. Como o schedule é único, os flows vão ser executados em sequência e no limite rodar por um certo periodo de tempo.

Problemas: se elas forem pesadas e a run for longa pode acontecer de rodar junto com outras pipelines pesadas e crashar o cluster. Como precaução eu configuraria o schedule pra rodar em blocos de 2 ou pipelines individuais.